### PR TITLE
Build v 2.7.0:

### PR DESCRIPTION
**Pull request by akabanovs**
- script updated.
 - dependencies updated.
 - description added to satisfy linter.
 JIRA ticket: https://anaconda.atlassian.net/browse/PKG-2534
 Upstream: https://github.com/coin-or/pulp
 CF: https://github.com/conda-forge/pulp-feedstock

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,22 +11,23 @@ source:
 build:
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     - rm "${SP_DIR}/pulp/solverdir/CoinMP.dll"  # [not win]
+  entry_points:
+    - pulptest = pulp.tests.run_tests:pulpTestAll
```
Comment (boldorider4) [inline recipe/meta.yaml:17]: wouldn't this be an entry point for tests?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,22 +11,23 @@ source:
...
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
-    - amply >=0.1.2
-    # We use glpk on Windows because coincbc is only packaged on Unix, currently.
+    - python
+    - amply
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: amply has been removed from the code https://github.com/coin-or/pulp/pull/403/files. There are no mentions in the source code already https://github.com/search?q=repo%3Acoin-or%2Fpulp+amply&type=pullrequests.
If we follow the conda-forge recipe can it be outdated?

Comment (boldorider4) [inline recipe/meta.yaml]: isn't amply an optional requirement for a plugin? I don't see it referenced in the code.

**Comment by skupr-anaconda** [comment]
Is it possible to run a test suite https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L82C5-L83C23 ? But I think `glpk` package will be required for testing https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L65

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Yes, I think you are right. My bad, I did not check well enough.

Comment (akabanovs) [reply recipe/meta.yaml]: Yeah, it seems it is redundant now. Thanks.

**Comment by akabanovs** [comment]
> Is it possible to run a test suite https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L82C5-L83C23 ? But I think `glpk` package will be required for testing https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L65

Yeah, I added pulptest entry and forgot to add it to the test commands. I think glpk is needed in tests, yeah. It skips less tests when I include glpk.

**Comment by skupr-anaconda** [comment]
> > Is it possible to run a test suite https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L82C5-L83C23 ? But I think `glpk` package will be required for testing https://github.com/coin-or/pulp/blob/8eb69cdf9845e0ad93f4b139f16e3fb8b73a2c08/INSTALL#L65
> 
> Yeah, I added pulptest entry and forgot to add it to the test commands. I think glpk is needed in tests, yeah. It skips less tests when I include glpk.

Your latest commit didn't run CI because artifacts exist on the staging channel

**Comment by skupr-anaconda** [comment]
An error on `s390x`:
```
conda.exceptions.ResolvePackageNotFound:
  - glpk
```
We can skip testing with glpk on `s390x`

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by boldorider4 — APPROVED** [review]
LGTM

**Pull request merged**